src/index.ts

```diff
@@ -17,8 +20,34 @@ function ensureDepIsInstalled(depName: string) {
   }
 }
 
+function ensurePeerDepRangeSatisfied(depName: string) {
+  try {
+    const installedVersion = require(`${depName}/package.json`).version
+
+    if (!installedVersion) {
+      return
+    }
+
+    const supportedRange = pkgJson.peerDependencies[depName]
```

Comment:
```suggestion
    const supportedRange = pkgJson.peerDependencies[depName]
    
    if (!supportedRange) {
      console.warn(
        console.red(`The package "nexus-plugin-prisma" has no such peer dep "${depName}" and so the check for if the consumer has satisfied that peer dep cannot be fulfilled.`
        )
      )
      return
    }
```

Since this entire block is inside a try-catch I opted to log here. Probably should be a throw though.